packages/amplify-graphql-api-construct-tests/src/commands.ts

```diff
@@ -84,6 +84,14 @@ export type CdkDeployProps = {
  * @returns the generated outputs file as a JSON object
  */
 export const cdkDeploy = async (cwd: string, option: string, props?: CdkDeployProps): Promise<any> => {
+  await spawn(getNpxPath(), ['cdk', 'bootstrap'], {
```

Comment:
We should not do this for all regions -- it's not clear to me what the behavior of a "bootstrap" is on a disabled optin region, but I have found instances of cdk deployment buckets for opt-in regions in accounts that do not have that region enabled.